pkg/trace/info/stats.go

```diff
@@ -155,15 +158,122 @@ func (ts *TagStats) publish() {
 	metrics.Count("datadog.trace_agent.receiver.events_extracted", eventsExtracted, tags, 1)
 	metrics.Count("datadog.trace_agent.receiver.events_sampled", eventsSampled, tags, 1)
 	metrics.Count("datadog.trace_agent.receiver.payload_accepted", requestsMade, tags, 1)
+
+	for reason, count := range ts.TracesDropped.tagValues() {
+		metrics.Count("datadog.trace_agent.normalizer.traces_dropped", count, append(tags, "reason:"+reason), 1)
+	}
+	for reason, count := range ts.SpansMalformed.tagValues() {
+		metrics.Count("datadog.trace_agent.normalizer.spans_malformed", count, append(tags, "reason:"+reason), 1)
+	}
+}
+
+// mapToString serializes the entries in this map into format "key1: value1, key2: value2, ...", sorted by
+// key to ensure consistent output order. Only non-zero values are included.
+func mapToString(statsMap map[string]int64) string {
```

Comment:
```suggestion
func mapToString(m map[string]int64) string {
```
It's just a `map`, not a "stats map" :)

Reply:
Updated